cli/test/fixtures/static-server.js

```diff
@@ -109,6 +109,12 @@ class Server {
         'Origin-Agent-Cluster': '?1',
       };
 
+      // This enables an important test in Smokerider - to check if the universal fetcher
+      // used for robots.txt (and other resources) is able to get around CORS.
```

Comment:
```suggestion
      // This enables an important test in Smokerider - to check if the universal fetcher
      // used for robots.txt (and source maps) is able to fetch freely while ignoring CORS constraints.
```

Reply:
but then.. the weird thing about this is... the robots.txt fetch is always same-origin (riiight?). 
so .. i kinda remember why this was a problem for sourcemaps
but... do we need cors-ignoring fetches for robots.txt? feels like no.

also, for sourcemaps, i'm prettttttty sure devtools frontend now defaults to fetching with same-origin restrictions and it seems to have not been a big problem (surprises me, but.. sure).

Anyway, now that you completely solved this nice lil bug, i'm thinking out loud that maybe our fetcher doesn't need cors-ignoring powers. 🤷‍♂️

Reply:
> Unless you mean…

yeah thats what i was getting at. i think were were attached to using a powerful fetcher but.. since then devtools frontend adopted fetching from the page's network context so.. I kinda think we may not need the power anymore.

But.. whatevs. Mostly documenting this for the next time we take a peek at the fetcher.